doc/src/explanation/active-deprecations.md

```diff
@@ -111,6 +111,10 @@ module. The code in ``sympy.physics.quantum`` currently does not ever
 return an ``IdentityOperator`` so the only place users will encounter
 its usage is in their own code.
 
+(deprecated-aesaraprinter)=
+### Deprecated aesaracode from printing
+The aesaracode.py in sympy.printing module has been deprecated due to unresolved dependency issues. It relies on outdated versions of NumPy and SciPy that are incompatible with the latest Python releases, making it unusable in current environments. 
```

Comment:
Long lines should be wrapped.

This should say that sympy's aesaracode module is deprecated because aesara itself is umaintained and cannot be installed on Python 3.13.